### PR TITLE
Update direct-routing-plan.md

### PR DESCRIPTION
**Pull request by NMuravlyannikov**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+The FQDNs – sip.pstnhub.gov.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.88.59
+- 52.127.92.64
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all the IP addresses above. 
+
+## SIP Signaling: Ports
+
+Prt requiremtns are the same for all Office 365 envirionments, where Direct Routing is offered:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Port requirements are the same for all Office 365 environments where Direct Routing is offered:
```

File: Teams/direct-routing-plan.md [hunk 2]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+- Office 365;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Office 365
```

File: Teams/direct-routing-plan.md [hunk 3]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+- Office 365 GCC;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Office 365 GCC
```

File: Teams/direct-routing-plan.md [hunk 4]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+- Office 365 GCC High;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Office 365 GCC High
```

File: Teams/direct-routing-plan.md [hunk 5]
```diff
@@ -210,9 +257,23 @@ The table below summarizes the relationships between primary, secondary, and ter
 ## Media traffic: Port ranges
 Note the requirements below apply if you ant to deploy Direct Routing without Media Bypass. For firewall requirements for Media Bypass please refer to [Plan for Media Bypass with Direct Routing](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-plan-media-bypass)
 
+
+
 The media traffic flows to and from a separate service in the Microsoft Cloud. The IP range for Media traffic:
+
+### Office 365 and Office 365 GCC envirionments
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
### Office 365 and Office 365 GCC environments
```

File: Teams/direct-routing-plan.md [hunk 6]
```diff
@@ -210,9 +257,23 @@ The table below summarizes the relationships between primary, secondary, and ter
...
+
 - 52.112.0.0 /14 (IP addresses from 52.112.0.1 to 52.115.255.254).
 
+### Office 365 GCC DoD envirionment
+
+- 52.127.64.0/21
+
+### Office 365 GCC High envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
### Office 365 GCC High environment
```

File: Teams/direct-routing-plan.md [hunk 7]
```diff
@@ -210,9 +257,23 @@ The table below summarizes the relationships between primary, secondary, and ter
...
+
+- 52.127.88.0/21
+
+### Port range (applicable to all envirionments)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
### Port range (applicable to all environments)
```

File: Teams/direct-routing-plan.md [hunk 8]
```diff
@@ -165,7 +165,17 @@ The certificate needs to be generated by one of the following root certificate a
 
 Microsoft is working on adding additional certification authorities based on customer requests. 
 
-## SIP Signaling: FQDNs and firewall ports 
+## SIP Signaling: FQDNs 
+
+Direct Routing is offered in the following Office 365 envirionments:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Direct Routing is offered in the following Office 365 environments:
```

File: Teams/direct-routing-plan.md [hunk 9]
```diff
@@ -165,7 +165,17 @@ The certificate needs to be generated by one of the following root certificate a
...
+- Office 365;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Office 365
```

File: Teams/direct-routing-plan.md [hunk 10]
```diff
@@ -165,7 +165,17 @@ The certificate needs to be generated by one of the following root certificate a
...
+- Office 365 GCC;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Office 365 GCC
```

File: Teams/direct-routing-plan.md [hunk 11]
```diff
@@ -165,7 +165,17 @@ The certificate needs to be generated by one of the following root certificate a
...
+- Office 365 GCC High;
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
- Office 365 GCC High
```

File: Teams/direct-routing-plan.md [hunk 12]
```diff
@@ -165,7 +165,17 @@ The certificate needs to be generated by one of the following root certificate a
...
+- Office 365 DoD
+
+More about GCC, GCC High and DoD envirionments [on this link] (https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Find out more about GCC, GCC High and DoD environments [on this link] (https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government).
```

File: Teams/direct-routing-plan.md [hunk 13]
```diff
@@ -165,7 +165,17 @@ The certificate needs to be generated by one of the following root certificate a
...
+
+### Office 365 and Office 365 GCC envirionments
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
### Office 365 and Office 365 GCC environments
```

File: Teams/direct-routing-plan.md [hunk 14]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
 - 52.114.7.24 
 - 52.114.14.70
 
-You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN sip-all.pstnhub.microsoft.com resolves to all the IP addresses above.  You must use the following ports:
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN sip-all.pstnhub.microsoft.com resolves to all the IP addresses above.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN sip-all.pstnhub.microsoft.com resolves to all these IP addresses. 
```

File: Teams/direct-routing-plan.md [hunk 15]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
 - 52.114.7.24 
 - 52.114.14.70
 
-You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN sip-all.pstnhub.microsoft.com resolves to all the IP addresses above.  You must use the following ports:
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN sip-all.pstnhub.microsoft.com resolves to all the IP addresses above. 
+
+
+### Office 365 GCC DoD envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
### Office 365 GCC DoD environment
```

File: Teams/direct-routing-plan.md [hunk 16]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+
+The connection point for Direct Routing is the following FQDN:
+
+**sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. Note as the Office 365 DoD envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
**sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. As the Office 365 DoD environment exists only in the US data centers, there is no secondary and tertiary FQDNs.
```

File: Teams/direct-routing-plan.md [hunk 17]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+
+The FQDNs – sip.pstnhub.dod.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.64.33
+- 52.127.68.34
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all the IP addresses above.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all these IP addresses. 
```

File: Teams/direct-routing-plan.md [hunk 18]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+**sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. Note as the Office 365 DoD envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs
+
+The FQDNs – sip.pstnhub.dod.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.64.33
+- 52.127.68.34
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all the IP addresses above. 
+
+### Office 365 GCC High envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
### Office 365 GCC High environment
```

File: Teams/direct-routing-plan.md [hunk 19]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+
+The connection point for Direct Routing is the following FQDN:
+
+**sip.pstnhub.gov.teams.microsoft.us** – Global FQDN. Note as the GCC High envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
**sip.pstnhub.gov.teams.microsoft.us** – Global FQDN. As the GCC High environment exists only in the US data centers, there is no secondary and tertiary FQDNs.
```

File: Teams/direct-routing-plan.md [hunk 20]
```diff
@@ -187,7 +197,44 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+
+The FQDNs – sip.pstnhub.gov.teams.microsoft.us will be resolved to one of the following IP addresses:
+
+- 52.127.88.59
+- 52.127.92.64
+
+You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all the IP addresses above.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all these IP addresses. 
```

File: Teams/direct-routing-plan.md [hunk 21]
```diff
@@ -210,9 +257,23 @@ The table below summarizes the relationships between primary, secondary, and ter
 ## Media traffic: Port ranges
 Note the requirements below apply if you ant to deploy Direct Routing without Media Bypass. For firewall requirements for Media Bypass please refer to [Plan for Media Bypass with Direct Routing](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-plan-media-bypass)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Note that the requirements below apply if you want to deploy Direct Routing without Media Bypass. For firewall requirements for Media Bypass, please refer to [Plan for media bypass with Direct Routing](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-plan-media-bypass).
```

File: Teams/direct-routing-plan.md [hunk 22]
```diff
@@ -210,9 +257,23 @@ The table below summarizes the relationships between primary, secondary, and ter
...
 
+
+
 The media traffic flows to and from a separate service in the Microsoft Cloud. The IP range for Media traffic:
+
+### Office 365 and Office 365 GCC envirionments
+
 - 52.112.0.0 /14 (IP addresses from 52.112.0.1 to 52.115.255.254).
 
+### Office 365 GCC DoD envirionment
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
### Office 365 GCC DoD environment
```

File: Teams/direct-routing-plan.md (hunk 12 above)
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
Learn more about [Office 365 and US Government environments](https://docs.microsoft.com/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government) such as GCC, GCC High, and DoD.
```

**Pull request merged**